corehq/apps/domain/dbaccessors.py

```diff
@@ -34,7 +34,8 @@ def get_doc_ids_in_domain_by_type(domain, doc_type, database=None):
     )]
 
 
-def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000, database=None):
+def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000,
```

Comment:
i think the odds of reuse would be low, but considering this returns ids, and you give it the starting id it'd be weird to think that the caller would need the starting id again